src/commands/dev/index.ts

```diff
@@ -56,6 +57,17 @@ export default async function main(ctx: NowContext) {
     return 2;
   }
 
+  const pkg = await readPackage();
+  if (pkg) {
+    // @ts-ignore
```

Comment:
Let's not skip type checking.

Instead, you can add the type to `type Package` like so:

```
type Package = {
  name: string;
  now?: Config;
  scripts?: { [key: string]: string }
};
```

Reply:
There's already a type in `src/util/dev/types.ts`, but `scripts` needs to be added to it.

Reply:
There's also a `PackageJson` type in `@now/build-utils`.

Reply:
@styfle @TooTallNate I used `src/util/dev/types.ts`, and added `scripts` there. 🙇‍♂️